spec/requests/vacancies_spec.rb

```diff
@@ -1,5 +1,74 @@
 require "rails_helper"
 
 RSpec.describe "Vacancies", type: :request do
-  pending "add some examples to (or delete) #{__FILE__}"
+  let(:user)    { build(:user) }
+  let(:vacancy) { build(:vacancy) }
+
+  describe "#index" do
+    it "returns http status success" do
+      get vacancies_path
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe "#new" do
+    before { sign_in user }
```

Comment:
if user is not company to redirect to root_page only test in code we will fix letter